### PR TITLE
Circuit laserpointers

### PR DESCRIPTION
**Pull request by TheBonded**
## About The Pull Request
Adds cool laserpointer modules to circuits

some notable differences between these and normal laserpointers:
- normal laserpointers stun cyborgs for 10-20 seconds, these laserpointers simply blind the cyborg for a fraction of a second without the stun
- normal laserpointers can make felinids dive for the light. this causes SEVERE time dilation increases when spammed, so I had to remove it from the circuits
- normal laserpointers can EMP cameras, circuit ones cannot for balance reasons
- normal laserpointers can blind people if shone in eyes, circuit laserpointers cannot target eyes and therefore can never blind people (I also didnt even code it in to save space)

## Why It's Good For The Game
circuits are cool, laserpointers are cool, and maybe people smarter than me can make cool logos with it or light shows
circuit laser logo bad

## Changelog

:cl:
expansion: Laserpointer circuit modules now available. they had to be toned down for circuit implementation, so dont expect them to do anything other than shine light.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
> Adds cool laserpointer modules to circuits

oh, coo-

> some notable differences between these and normal laserpointers:

normal laserpointers stun cyborgs for 10-20 seconds, these laserpointers simply blind the cyborg for a fraction of a second without the stun
normal laserpointers can make felinids dive for the light. this causes SEVERE time dilation increases when spammed, so I had to remove it from the circuits
normal laserpointers can EMP cameras, circuit ones cannot for balance reasons
normal laserpointers can blind people if shone in eyes, circuit laserpointers cannot target eyes and therefore can never blind people (I also didnt even code it in to save space)

aw

**Comment by TheBonded** [comment]
> aw

@ATH1909 lets be real none of those would ever be allowed anyway even if I did code them in, circuits are not allowed to have any combat capabilities 

at least you can make cool light shows and make cyborgs panic for a second

**Comment by moocowswag** [comment]
whats the cooldown on laser pointer.

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -254,6 +254,7 @@
 		"comp_index_assoc",
 		"comp_index_table",
 		"comp_length",
+		"comp_laserpointer",
```
Comment (Watermelon914) [inline code/modules/research/techweb/all_nodes.dm:257]: alphabetical, should be before comp_length

Comment (TheBonded) [reply code/modules/research/techweb/all_nodes.dm:257]: missed that, sorry

**Review by Watermelon914 — APPROVED** [review]

**Pull request merged**